### PR TITLE
update Java9

### PR DESCRIPTION
**Pull request by Rsplwe**

### CODE REVIEW

**Comment by SupremeMortal** [comment]
Why have you deleted ExceptionHandler and BugReportGenerator? Also, don't you think it would have been easier to test this with an IDE instead of editing the files directly on GitHub.

**Comment by Rsplwe** [comment]
我在学校的电脑上没有IDE和Git Bash所以只能在Github在线编辑文件

**Comment by sero583** [comment]
@SupremeMortal He was probably coding on a phone, I know that worse experience.

**Comment by ScraM-Team** [comment]
I doubt most people are ready to move to Java 9 yet.  Probably many hosts don't support it yet either.  I would like to wait, personally.

File: src/main/java/module-info.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module nukkit {
+	requires java.xml.bind;
```
Comment (liach) [inline src/main/java/module-info.java]: tab和空格别混用

Comment (Rsplwe) [reply src/main/java/module-info.java]: 没有混用，github的识别问题。。。

**Pull request closed without merging**